src/HornedBeast.js

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+ 
+
+    class HornedBeast extends Component {
+        render(){
+            return(
+                <div>
+                    <h2>{this.props.title}</h2>
+                    <img src={this.props.img_url} alt={this.props.alt} title={this.props.title} />
+
+                    <p>{this.props.description}</p> 
+
+                </div>
```

Comment:
Really good work so far!

Feedback here is for spacing. We can just have the <p> tag directly below the <img> tag.